lib/ziti_tunnel.c

```diff
@@ -110,8 +117,53 @@ int ziti_tunneler_host_v1(tunneler_context tnlr_ctx, const void *ziti_ctx, const
     return 0;
 }
 
+static void send_dns_resp(uint8_t *resp, size_t resp_len, void *ctx) {
+    struct resolve_req *rreq = ctx;
+
+    ZITI_LOG(INFO, "sending DNS resp[%zd] -> %s:%d", resp_len, ipaddr_ntoa(&rreq->addr), rreq->port);
+    struct pbuf *rp = pbuf_alloc(PBUF_TRANSPORT, resp_len, PBUF_RAM);
+    memcpy(rp->payload, resp, resp_len);
+
+    err_t err = udp_sendto_if_src(rreq->tnlr_ctx->dns_pcb, rp, &rreq->addr, rreq->port,
```

Comment:
Why use udp_sendto_if_src here? To specify the src IP? I would udp_sendto could work since dns_pcb should be bound to the right IP anyway.